config/initializers/allowed_request_origins.rb

```diff
@@ -3,7 +3,7 @@
 #
 #   export ALLOWED_REQUEST_ORIGINS=https://myhost.com,https://myotherhost.com
 #
-if (allowed_request_origins = ENV['ALLOWED_REQUEST_ORIGINS'].to_s.split(',')).any?
```

Comment:
I tried changing this to `Setting.allowed_request_origins`, but `Setting` isn't loaded at the time this file is run. I'd expected it to be auto-loaded, but I'm not sure it's safe to do so, or even desired. Thoughts?

Reply:
Ohh, hm, that kind of makes sense b/c initializers must run early. In that case let’s just leave it as is. We have a few other “deep” things like DATABASE_NAME and stuff like that which are ENV only so maybe there is a clear distinction between server level settings and app-level settings.